### PR TITLE
[13.0][MIG] hr_holidays_notify_employee_manager

### PR DESCRIPTION
**Pull request by watcharaporn-git**
@saran440 

This module extends the functionality of the Leaves App to make possible
to notify by mail the manager of the employee requesting the leave.

### CODE REVIEW

File: hr_holidays_notify_employee_manager/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "HR Holidays Notify Employee Manager",
+    "summary": "Notify employee's manager by mail on Leave Requests " "creation.",
+    "version": "13.0.1.1.0",
```
Comment (kittiu) [inline hr_holidays_notify_employee_manager/__manifest__.py]:
```suggestion
    "version": "13.0.1.0.0",
```

File: hr_holidays_notify_employee_manager/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "HR Holidays Notify Employee Manager",
+    "summary": "Notify employee's manager by mail on Leave Requests " "creation.",
+    "version": "13.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/hr",
```
Comment (ps-tubtim) [inline hr_holidays_notify_employee_manager/__manifest__.py:8]:
```suggestion
    "website": "https://github.com/OCA/hr-holidays",
```
Please change the website.

File: hr_holidays_notify_employee_manager/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
+* Lois Rilo <lois.rilo@eficent.com>
+* Miquel Raïch <miquel.raich@eficent.com>
+* Dennis Sluijk <d.sluijk@onestein.nl>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
+* Nikul Chaudhary <nikul.chaudhary.serpentcs@gmail.com>
+* Alexey Pelykh <alexey.pelykh@brainbeanapps.com>
```
Comment (ps-tubtim) [inline hr_holidays_notify_employee_manager/readme/CONTRIBUTORS.rst:6]: Please add you as a contributor.

**Comment by Saran440** [comment]
This module already in core odoo.

**Pull request closed without merging**